vllm/distributed/kv_transfer/kv_connector/v1/base.py

```diff
@@ -185,21 +211,19 @@ def wait_for_save(self):
         """
         pass
 
-    def get_finished(
-        self, finished_req_ids: set[str]
-    ) -> tuple[Optional[set[str]], Optional[set[str]]]:
+    def build_worker_events(
+        self, model_runner_output: ModelRunnerOutput
```

Comment:
Why change what is passed here. There are cases that might need the finished requests.

What specifically might the ModelRunnerOutput be needed for in this context?

Reply:
`SchedulerOutput` contains `finished_req_ids`.
My original implementation passed `SchedulerOutput` to `build_worker_events`.
@sdavidbd suggested that I remove `SchedulerOutput` from this function.
See here:
https://github.com/vllm-project/vllm/pull/19555#discussion_r2154958436

`finished_req_ids` can still be accessed by the worker, but it needs to be encoded by the scheduler-side connector on `build_connector_meta`.